magento/pwa-studio
# AC-2478::Search results summary information not announced to screen

## Description

**Reproduction Steps**
[NODE][body>div:nth-of-type(1)>*:nth-child(1)>*:nth-child(2)]

1. With JAWS operating, press Tab to move through the content.
2. When keyboard focus is on Search, press Enter or Space to activate it.
3. Enter value to search on and press enter.

**Actual Behavior**
Screen reader users are not notified when "No results were found" or number of results returned for search.

When developers don't specify which portions of a page are live regions, screen readers and other assistive technologies will not know which areas of the page to monitor for updates.

**Expected Behavior**
Ensure screen readers are notified of summary of the number of search results returned using aria-live="polite".

## Related Issue







Closes https://jira.corp.magento.com/browse/AC-2478

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps
Pre-Conditions:
1. Have Magento instance with sample data installed
2. Make sure to have pwa studio installed
3. Make sure to have a customer login for front end login

**Manual Steps executed:**

Enable screen reader extension
navigate to serach button and serach any product 
observe the response by screen reader when No results were found for the serached product 
observe the response by screen reader when number of results returned for searched product{}

**✖️Behaviour Before The Fix :**

Screen reader users are not notified when "No results were found" or number of results returned for search.

**✔️Behaviour After The Fix:** 

Screen reader users are  notified when "No results were found" or number of results returned for search.







## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
